kalite/distributed/features/steps/learn_content.py

```diff
@@ -3,7 +3,7 @@
 from kalite.testing.behave_helpers import build_url, reverse, alert_in_page, find_css_class_with_wait, \
     assert_no_element_by_css_selector
 
-TIMEOUT = 3
+TIMEOUT = 30
```

Comment:
I think 30 seconds timeout mixed with `assert_no_element_by_css_selector` sounds slow, but I'll create a separate issue for it.

You are more than welcome to work on #4900 if you feel like it :) You don't need to solve all of it in one PR, but any changes that can help us optimize testing are very welcomed!!